docs/setup.md

```diff
@@ -1,44 +1,61 @@
 | [Home](../README.md) |
-|--------------------------------------------|
+|----------------------|
 
 # Installation
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Continuous Delivery**. 
+2. From the list of widgets that appears, search for and select **CICD Configuration Wizard**. 
 
-3. Click the card of the **Continuous Delivery** solution pack card.
+3. Click the card of the **CICD Configuration Wizard** widget card.
 
-4. Click **Configure** on the bottom to begin configuration.
+4. Click **Install** on the bottom to begin installation.
 
 # Configuration
 
 ## CICD Configuration Wizard Widget Settings
 
 The configuration settings of the CICD Configuration widget include the following steps to setup CICD configuration:
 
-- Click the **Configure** button to bring up the following wizard screen
+1. Click the **Configure** button to bring up the following wizard screen.
 
-    ![CICD Configration](./res/cicd_configuration.png)
+    ![CICD Configuration](./res/cicd_configuration.png)
 
-- Click **Let's Get Started** to proceed
+2. Click **Let's Get Started** to proceed.
 
-- On **Define Environment** page, select the FortiSOAR environment type &mdash; **Development** and **Production**.
+3. **Define Environment**
 
-- Click **Next** on lower-right corner
+    1. Select the FortiSOAR environment type &mdash; **Development** and/or **Production**.
 
-    ![Define Environment](./res/define_environment.png)
+    2. Click **Next** on lower-right corner to proceed.
 
-- On the **Connect Version Control** page, configure your GitHub version control environment.
+        ![Define Environment](./res/define_environment.png)
 
-- Click **Next** on lower-right corner
+4. **Define Your Source Control**
 
-    > **Note:** For now, only version control using GitHub is supported
+    1. Select the source control to configure. You can select from following options:
+        1. GitHub
+        2. GitLab
 
-    ![Connect Version Control](./res/connect_version_control.png)
+    2. Provide the base URL of the source control.
 
-- Click **Finish** to complete the configuration.
+    3. Click **Next** on lower-right corner to proceed.
+
+        ![Select Source Control](./res/select_source_control.png)
+
+5. **Connect Version Control**
+
+    1. Specify the selected source control's configuration details.
+
+    2. Click **Next** on lower-right corner to proceed.
+
+        ![Connect Version Control](./res/connect_version_control.png)
+
+- Click **Next** to complete the configuration.
 
     ![Finish Configuration](./res/finish_configuration.png)
```

Comment:
@cs-abhishek-shukla : This screenshot will also change. please connect with Sujit for the same.

Reply:
Done